style.css

```diff
@@ -0,0 +1,815 @@
+    height: 814px;
+    letter-spacing: 5px;
+  }
+
+  .text__title {
+    position: inherit;
+    align-items: flex-end;
+    font-size: 86px;
+    line-height: 92px;
+    padding-left: 52px;
+    letter-spacing: 2px;
+  }
+
+  .vectormob {
+    position: absolute;
+    width: 65px;
+    height: 37px;
+    left: 300px;
+    top: 873px;
+    display: block;
+  }
+}
+
+@media (max-width: 565px) {
+  .first-container {
+    padding: 189px 0;
+    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+    background-size: 565px;
+    background-position: center;
+  }
+
+  .header__inner {
+    width: 100%;
+    height: 104px;
+    background: #fff;
+    z-index: 2;
+  }
+
+  .header__menu {
+    background: #EDEDED;
+    z-index: 3;
+    height: 104px;
+    width: 104px;
+  }
+
+  .header__menu-item {
+    height: 4px;
+    width: 42.7px;
+    margin-bottom: 7px;
+  }
+
+  .header__logo p {
+    margin: 15px 9px;
+    color: black;
+    font-size: 45px;
+    letter-spacing: 4px;
+  }
+
+  .text__container {
+    display: flex;
+    align-items: flex-end;
+    width: 100%;
+    max-width: 750px;
+    height: 677px;
+    letter-spacing: 5px;
+  }
+
+  .text__title {
+    position: inherit;
+    align-items: flex-end;
+    font-size: 73px;
+    line-height: 80px;
+    padding-left: 43px;
+    letter-spacing: 3px;
+  }
+
+  .vectormob {
+    position: absolute;
+    width: 55px;
+    height: 30px;
+    left: 45.5%;
+    top: 734px;
+    display: block;
+  }
+}
+
+@media (max-width: 490px) {
+  .first-container {
+    padding: 127px 0;
+    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+    background-size: 490px;
+    background-position: center;
+  }
+
+  .header__inner {
+    width: 100%;
+    height: 89px;
+    background: #fff;
+    z-index: 2;
+  }
+
+  .header__menu {
+    background: #EDEDED;
+    z-index: 3;
+    height: 89px;
+    width: 89px;
+  }
+
+  .header__menu-item {
+    height: 4px;
+    width: 36.7px;
+    margin-bottom: 7px;
+  }
+
+  .header__logo p {
+    margin: 15px 4px;
+    color: black;
+    font-size: 38px;
+    letter-spacing: 4px;
+  }
+
+  .text__container {
+    display: flex;
+    align-items: flex-end;
+    width: 100%;
+    max-width: 750px;
+    height: 569px;
+    letter-spacing: 5px;
+  }
+
+  .text__title {
+    position: inherit;
+    align-items: flex-end;
+    font-size: 63px;
+    line-height: 67px;
+    padding-left: 36px;
+    letter-spacing: 3px;
+  }
+
+  .vectormob {
+    position: absolute;
+    width: 51px;
+    height: 25px;
+    left: 45%;
+    top: 624px;
+    display: block;
+  }
+}
+
+@media (max-width: 422px) {
+  .first-container {
+    padding: 71px 0;
+    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+    background-size: 422px;
+    background-position: center;
+  }
+
+  .header__inner {
+    width: 100%;
+    height: 80px;
+    background: #fff;
+    z-index: 2;
+  }
+
+  .header__menu {
+    background: #EDEDED;
+	  z-index: 3;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)